specification/recoveryservicessiterecovery/resource-manager/Microsoft.RecoveryServices/stable/2022-02-01/service.json

```diff
@@ -24032,6 +24032,10 @@
         "targetDiskName": {
           "description": "The target disk name.",
           "type": "string"
+        },
+        "isOSDisk": {
+          "description": "A value indicating whether the disk is the OS disk.",
+          "type": "string"
```

Comment:
the intent of the "avoid booleans" linter is not to change everything to string.  String values of "true"/"false" are confusing to users.  The suggestion is instead of boolean use something like "diskType": "OS"/"Data"/"Whatever"

Reply:
this should also be a string enum (see other uses of "x-ms-enum" in this file)

Reply:
Hi @pilor ,
We have been using this convention everywhere since starting of this service. Everywhere we are using same format. Same value is already being returned in GET API in same format. Few examples are: 
1. https://github.com/Azure/azure-rest-api-specs/blob/4611eaa0300ed93a6f9b59db3119cad2b350d4e7/specification/recoveryservicessiterecovery/resource-manager/Microsoft.RecoveryServices/stable/2022-02-01/service.json#L9848
2. https://github.com/Azure/azure-rest-api-specs/blob/4611eaa0300ed93a6f9b59db3119cad2b350d4e7/specification/recoveryservicessiterecovery/resource-manager/Microsoft.RecoveryServices/stable/2022-02-01/service.json#L15603

We would like to maintain the same format in every API and not change right now.